🐛 Remove the OPENEXR_INSTALL_EXAMPLES CMake option

Jamim opened the pull request:
_Hello @cary-ilm,_

The `OPENEXR_INSTALL_EXAMPLES` option is misleading and doesn't have any effect.
It might be safely removed in favor of the `OPENEXR_BUILD_EXAMPLES` option.

_Best regards!_

cary-ilm commented on website/install.rst:
```diff
@@ -460,7 +460,7 @@ Component Options
-* ``OPENEXR_INSTALL_EXAMPLES``
+* ``OPENEXR_BUILD_EXAMPLES``
 
   Build and install the example code. Default is ``ON``.
```
Should remove any reference to install here:
```suggestion
  Build the example code. Default is ``ON``.
```

Jamim replied:
Thank you! I've submitted an updated commit.

cary-ilm reviewed (changes requested):
One small suggestion below, but otherwise perfect, thanks!  Last time I looked I suspected that was superfluous but didn't do anything about it.

Jamim committed:
🐛 Remove the OPENEXR_INSTALL_EXAMPLES CMake option

The OPENEXR_INSTALL_EXAMPLES option is misleading and doesn't have any effect.
It might be safely removed in favor of the OPENEXR_BUILD_EXAMPLES option.

Co-authored-by: Cary Phillips <cary@ilm.com>
Signed-off-by: Aliaksei Urbanski <aliaksei.urbanski@gmail.com>

cary-ilm reviewed (approved):
Thanks!
